src/api/db/migrate/20211110142710_create_workflow_runs.rb

```diff
@@ -0,0 +1,15 @@
+class CreateWorkflowRuns < ActiveRecord::Migration[6.1]
+  def change
+    create_table :workflow_runs, id: :integer do |t|
+      t.text :request_headers
+      t.text :request_payload
+      t.string :status
+      t.text :response
+      t.string :response_url
+
+      t.timestamps
+
+      t.belongs_to :token, index: true, type: :integer
```

Comment:
A workflow run should exist only if it's linked to a token, so `token_id` cannot be `null`:

```suggestion
      t.belongs_to :token, index: true, type: :integer, null: false
```

Reply:
Right. Change applied.